Quiz

VikingPaul opened the pull request.

VikingPaul committed:
framework and bootstrap created

VikingPaul committed:
JSON file made

VikingPaul committed:
.js files dealt with

VikingPaul committed:
Made minor changes.
Burn with fire working.

VikingPaul committed:
navbar changed to take up full length of window

VikingPaul committed:
fixed bug
bug would change description when typing after using the burn with fire
button.

zoeames commented:
Hey paul looks great!!!  Just a couple minor things
1.  navbar - make it more bootstrappy.  you will find that bootstrap navbars seem to have a lot of what looks like unnecessary code but most of that adds a lot of padding and stuff. 
2.  your code looks very clean but I would like to see you use a LOT more ES6 where you can - replace var with let etc.
3. Take a closer look at these instructions :  ' Each row should contain 3 columns. Make sure you have a parent element with a class of container'

you will want to make use of the bootstrap row class here.  also you should have a parent element with class container - currently yours has a class and id of placement

VikingPaul committed:
Edits for Zoe:
The navbar looks more bootstrappy. Added classes to the navbar.
More ES6. Added "use strict" to JS and replaced vars with lets and put in
fat arrow.
Fixed class to container.

zoeames commented:
now we are talking!  Looks great!  merge it in
👍
